proc_CPMG.py

```diff
@@ -4,26 +4,39 @@
 from proc_scripts import postproc_dict
 from proc_scripts.fitting import decay
 fl = fl_mod()
-logger = init_logging('info')
+logger = init_logging('debug')
+dwdel1=6.5e-6
+TD=4
+tau_extra=20e-6
 for searchstr, exp_type, nodename, postproc, label_str, f_range in [
         #('200221_CPMG_TEMPOLgel_2p9_1','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        #('200304_CPMG_2p6_1','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
+        #('w8_200731','test_equip',3,'ag_CPMG_strob','water loading 8',(-500,500)),
+        ('200304_CPMG_2p6_1','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
         #('200305_CPMG_3p5_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
         #('200305_CPMG_3p6_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        ('200305_CPMG_3p7_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
+        #('200305_CPMG_3p7_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
         #('200305_CPMG_3p7_3','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
         #('200305_CPMG_3p8_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
         #('200305_CPMG_3p9_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
         #('200305_CPMG_4p0_1','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
         ]:
-    s =  find_file(searchstr, exp_type=exp_type,
+    if postproc=='spincore_CPMG_v1':
+        s =  find_file(searchstr, exp_type=exp_type,
             expno=nodename, postproc=postproc, lookup=postproc_dict)
-    #{{{ centering CPMG echo
-    s = center_CPMG_echo(s)
+    if postproc=='ag_CPMG_strob':
+        s = find_file(searchstr, exp_type=exp_type,
+                expno=nodename, postproc=postproc,
+                lookup=postproc_dict,fl=fl)
+        #{{{ centering CPMG echo
```

Comment:
shouldn't have postprocessing-dependent if statements.  In fact, if I compare the two commands, they are the same (except one is plotting and the other is not).